lib/_debugger.js

```diff
@@ -1639,7 +1639,16 @@ Interface.prototype.trySpawn = function(cb) {
   } else if (this.args.length === 3) {
     // `node debug -p pid`
     if (this.args[1] === '-p' && /^\d+$/.test(this.args[2])) {
-      process._debugProcess(parseInt(this.args[2], 10));
+      const pid = parseInt(this.args[2], 10);
+      try {
+        process._debugProcess(pid);
+      } catch (e) {
+        if (e.code === 'ESRCH') {
```

Comment:
I have a question to reviewers.

`process.bindings('uv')` has some error definitions like ESRCH.
so we can write the following code : 

```javascript
const uv = process.bindings('uv');

if (e.code === uv.errname(uv.UV_ESRCH)) {
  // ...
}
```

is this better way? if any reviewer has some opinions, please tell me.
we can prevent some typos like `ESCRH`.